docs/fluxdoc.md

```diff
@@ -0,0 +1,227 @@
+# fluxdoc formatting
+
+`fluxdoc` uses comments in `.flux` package files in `/stdlib` to generate and
+output standard library documentation in JSON format.
+The generated JSON is used to build the public-facing Flux standard library
+documentation and ensure documentation is up-to-date and featur-complete with
+each new Flux release.
+
+**Note:** Packages are required to pass the `fluxdoc` linter before being merged
+into `master` unless listed as an exception in
+[`/libflux/flux-core/src/bin/fluxdoc.rs`](https://github.com/influxdata/flux/blob/master/libflux/flux-core/src/bin/fluxdoc.rs).
```

Comment:
Should we not mention the exception list? We intent to completely remove that list once its empty. As this reads now it sort of implies you could add packages to that list which we do not want to encourage.

Reply:
Removed.